modules/cluster-logging-collector-legacy.adoc

```diff
@@ -0,0 +1,197 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-legacy_{context}"]
+= Forwarding logs using the legacy configuration method
+
+You can configure cluster logging to send logs to destinations outside of your {product-title} cluster instead of the default Elasticsearch log store using Fluentd or syslog by creating a configuration file and ConfigMap.
+
+[IMPORTANT]
+====
+These methods for forwarding logs are deprecated in {product-title} and will be removed in a future release.
+====
+
+Sending logs using the Fluentd forward protocol::
+
+You can use the Fluentd *forward* protocol to send a copy of your logs to an external log aggregator configured to accept the protocol, instead of the default Elasticsearch log store. You are responsible for configuring the external log aggregator to receive the logs from {product-title}. 
+
+ifdef::openshift-origin[]
+The *forward* protocols are provided with the Fluentd image as of v1.4.0.
+endif::openshift-origin[]
+
+To configure {product-title} to send logs using the Fluentd *forward* protocol, create a configuration file, called `secure-forward.conf`, that points to an external log aggregator. Then use that file to create a ConfigMap called called `secure-forward` in the `openshift-logging` namespace, which {product-title} uses when forwarding the logs. 
+
+.Sample Fluentd configuration file
+
+[source,yaml]
+----
+<store>
+  @type forward
+  <security>
+    self_hostname fluentd.example.com 
+    shared_key "fluent-receiver"
+  </security>
+  transport tls
+  tls_verify_hostname false
+  tls_cert_path '/etc/ocp-forward/ca-bundle.crt'
+  <buffer>
+    @type file
+    path '/var/lib/fluentd/secureforwardlegacy'
+    queued_chunks_limit_size "1024"
+    chunk_limit_size "1m"
+    flush_interval "5s"
+    flush_at_shutdown "false"
+    flush_thread_count "2"
+    retry_max_interval "300"
+    retry_forever true
+    overflow_action "exception"
+  </buffer>
+  <server>
+    host fluent-receiver.example.com
+    port 24224
+  </server>
+</store>
+----
+
+Sending logs using syslog:: 
```

Comment:
@mburke5678  could you please update this part from line 57 to 88? 
With the recent changes syslog supports TLS and both RFC 3164 & 5424. 
For ref: https://github.com/openshift/cluster-logging-operator/blob/master/manifests/4.6/logging.openshift.io_clusterlogforwarders_crd.yaml#L152